bundles/org.openhab.binding.adorne/README.md

```diff
@@ -0,0 +1,113 @@
+# Adorne Binding 
+
+The Adorne Binding integrates [Adorne Wi-Fi ready devices](https://www.legrand.us/adorne/products/wireless-whole-house-lighting-controls.aspx) (switches, dimmers, outlets) from [Legrand](legrand.com).
+
+Legrand attempted to provide a public API based on Samsung's ARTIK Cloud and the initial version of this binding was based on that API.
+However, Samsung shut down ARTIK Cloud shortly after the release and Legrand has not offered a public API replacement since.
+That leaves direct interaction with the Adorne Hub as the only control option.
+Consequently the OpenHAB server and the Adorne Hub must be located on the same network.
+
+The Adorne Hub supports a REST API, but unfortunately there is no documentation or official support from Legrand.
+This binding's implementation of the REST API is motivated by the great work of [sbozarth](https://github.com/sbozarth/homebridge-lc7001) who figured out the API details.
+
+## Supported Things
+
+| Thing Type|Description|
```

Comment:
Please format the tables with https://www.tablesgenerator.com/markdown_tables

Reply:
Done